docs/faq.md

```diff
@@ -45,6 +45,16 @@ the tutorial [here|Testing-new-strategies-with-Hyperopt](bot-usage.md#hyperopt-c
 
 You can use the `/forcesell all` command from Telegram.
 
+### I get the message "RESTRICTED_MARKET"
+
+Currently known to happen for US Bittrex users.  
+Bittrex split it's exchange into an US and an International verssion.
+The International version has more pairs available, however the API always returns all pairs, so there is currently no automated way to detect if you're affected by the restriction.
+
+If you have restricted pairs in your whitelist, you'll get a Warning-message on startup for each restricted pair.
+If you're an "International" Customer on the Bittrex exchange, then this warning will not impact you.
```

Comment:
"will probably not impact"
(?)

Reply:
yes, it's probably (it assumes bittrex recognizes the user/bot correctly as international ...